### ISSUE TITLE
Count combinations returns NAN

### ISSUE THREAD

**Issue by dmitry-zaporozhan** [drupol/phpermutations#10]
Consider using bcmath in factorial calculations.
This code results in `NAN`
```
$p = new Combinations(range(1, 200), 2);
var_dump($p->count());
```
While this code results in `19900`
```
$p = new Combinations(range(1, 200), 2);
var_dump(count($p->toArray()));
```

**Comment by drupol** [comment]
thanks, will check this out asap.

**Comment by drupol** [comment]
This is indeed a confirmed bug.

I'm still wondering if I should use `count($p->toArray())` to get the count or rely on a mathematical formula.
The issue while using the first proposition is that it needs to compute the whole solutions array.
The issue using the last proposition is that it might generate huge numbers and then they gets simplified (_root of the issue you reported_).

I'm still wondering what to do.

**Comment by drupol** [comment]
Fixed, tagged a new version 1.3.3 ready for use.

Thanks for the bug report!
